tests/test_round_trip.rs

```diff
@@ -11,6 +11,19 @@ fn roundtrip(original: &Yaml) {
     assert_eq!(documents[0], *original);
 }
 
+fn roundtrip_multiline(original: &Yaml) {
+    let mut emitted = String::new();
+    let mut emitter = YamlEmitter::new(&mut emitted);
+    emitter.multiline_strings(true);
+    emitter.dump(original).unwrap();
+
+    let documents = Yaml::load_from_str(&emitted).unwrap();
+    println!("emitted {emitted}");
```

Comment:
was this a leftover debugging println or is this here for future debugging purposes? just checking in case we should trim it.

Reply:
I am not against keeping it anyway. They are captured by `cargo test` and discarded unless the test fails.

Our test suite takes ~0.2s on my machine to run so we can afford a little bit of extra debugging detail should something fail in the future.